src-tauri/src/config/server_connection.rs

```diff
@@ -45,6 +55,7 @@ impl Default for ServerConnection {
             ip_address: "".to_string(),
             ssh_public_key,
             ssh_private_key,
+            user: "root".to_string(),
```

Comment:
Is this going to pop up OS permission dialogs?

Reply:
This is only to reduce the number of places we were repeating the same ssh information